src/Analyzer.BicepProcessor/Analyzer.BicepProcessor.csproj

```diff
@@ -9,8 +9,10 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Azure.Bicep.Core" Version="0.8.9" />
+    <PackageReference Include="Azure.Bicep.Core" Version="0.11.1" />
+    <PackageReference Include="Azure.Bicep.Types" Version="0.3.28" /> <!-- temp -->
```

Comment:
Can we already remove these temp references?

Reply:
[New Bicep.Core](https://www.nuget.org/packages/Azure.Bicep.Core/0.12.1) released yesterday so should be able to!